mobile-workspace/latest/using/index.md

```diff
@@ -154,32 +153,82 @@ Filters can be set for Status, Due date, and Task Name. Filter chips are shown a
 
 2. Follow the directions for each filter as follows:
 
-    * A Status filter can be set by selecting “Active” or “Completed” and press **Apply**. The Status filter chip label will show the active filter, so “Completed” instead of “Status”.
-    * Press **Due date** to set a custom, date-ranged filter. “From” and “To” start and end fields set using a calendar date picker. Only one date needs to be set.
+    * A Status filter can be set by selecting **Active** or **Completed** and then press **Apply**. The Status filter chip label will show the active filter, so “Completed” instead of “Status”.
+    * Press **Due date** to set a custom, date-ranged filter. **From** and **To** start and end fields are set using a calendar date picker. Only one date needs to be set.
     * Press **Task Name** to filter tasks by name. Type your search in a text field. This filter does not apply to priority and is not case sensitive.
 
-3. Press **Apply** to set the filter and **Reset** to reset the filter to default. The **X** in the top right of the filter UI will close the window and return to the Task list.
+3. Press **Apply** to set the filter and **Reset** to reset the filter to default. The **X** in the top right closes the window and returns you to the Task list.
 
 4. The filter chips will show the active filters and the refresh button will reset the filters.
 
 ### Task view
 
-Within the Task view are details about the task: Task name, Due date, Priority, Assignee, Status, and Identifier. There is a back arrow to exit Task view and return to the Task list.
+Within the Task view are details about the task: Task name, Due date, Priority, Assignee, Status, and Identifier. There is a back arrow to exit the Task view and return to the Task list.
 
-Individual tasks have a “Comments” section below the task details. The most recent comment is shown with the user's icon, user name, date the comment was added, and the user’s comment. If there is more than one comment, you will see a link to **View all**. The link **Add a comment…** is below existing comments.
+Individual tasks have a Comments section below the task details. The most recent comment is shown with the user's icon, user name, date the comment was added, and the user’s comment. If there is more than one comment, you will see a link to **View all**. The link **Add a comment…** is below the existing comments. If you have access to  
```

Comment:
not sure what is ' if you have access to ...' no full stop or did you intend to continue this ?

Reply:
Deleted, thanks.